src/styles/caniuse.css.js

```diff
@@ -9,7 +9,6 @@ export default css`
   flex-wrap: wrap;
   justify-content: flex-start;
   align-items: baseline;
-  cursor: pointer;
```

Comment:
Should probably use `cursor: default` instead of pointer or the default `text`?

Reply:
checking...

Reply:
`text` is no good and it's the text selection cursor. And `cursor: default` is the same as none.